action.yml

```
@@ -24,9 +24,10 @@ inputs:
     required: false
     default: 'false'
   python_version:
-    description: 'Python version to install'
+    description: 'Python version to use for Conan installation'
```

```suggestion
    description: 'Python version to install'
```
why changing the description if we are deprecating this?